chart/ohmyglb/values.yaml

```diff
@@ -0,0 +1,65 @@
+global:
+  ## Reference to one or more secrets to be used when pulling images
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  imagePullSecrets: []
+  # - name: "image-pull-secret"
+
+ohmyglb:
+  image: ytsarev/ohmyglb:v0.2.1
```

Comment:
Similar to Helm version, reset on first release?

Reply:
@donovanmuller we can, but reset to what ? `v0.2.1` btw reflects semantics of actual development